webpack-test.config.js

```diff
@@ -0,0 +1,52 @@
+const webpack = require( 'webpack' );
```

Comment:
Hmm I don't agree, in karma should be just karma config, this makes it more cleaner IMHO